docs/app/Examples/elements/Image/Variations/index.js

```diff
@@ -1,43 +1,42 @@
 import React from 'react'
 import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
 import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
-import { Message } from 'semantic-ui-react'
 
 const ImageVariationsExamples = () => (
   <ExampleSection title='Variations'>
     <ComponentExample
       title='Avatar'
-      description='An image can appear as an Avatar.'
+      description='An image may be formatted to appear inline with text as an avatar.'
       examplePath='elements/Image/Variations/ImageExampleAvatar'
     />
     <ComponentExample
       title='Bordered'
-      description='An image can be bordered.'
+      description='An image may include a border to emphasize the edges of white or transparent content.'
       examplePath='elements/Image/Variations/ImageExampleBordered'
     />
     <ComponentExample
       title='Fluid'
-      description='A fluid image has no maximum width.'
+      description='An image can take up the width of its container.'
       examplePath='elements/Image/Variations/ImageExampleFluid'
     />
     <ComponentExample
-      title='Shapes'
-      description='An image can have different shapes.'
+      title='Rounded'
+      description='An image may appear rounded.'
       examplePath='elements/Image/Variations/ImageExampleRounded'
     />
-    <ComponentExample examplePath='elements/Image/Variations/ImageExampleCircular'>
-      <Message warning>
-        Perfectly circular images require a perfectly square image file.
-      </Message>
```

Comment:
I think we should leave this warning in here.  I lost quite a bit time trying to debug why our images were oval and SUI core was circular.  Awkwardly enough, the `circular` class does _not_ make circular images, it makes "fully rounded" images if you will.  There is a similar warning in the SUI core docs.